src/m365/teams/commands/chat/chat-get.ts

```diff
@@ -0,0 +1,200 @@
+  name?: string;
+}
+
+class TeamsChatGetCommand extends GraphCommand {
+  public get name(): string {
+    return commands.CHAT_GET;
+  }
+
+  public get description(): string {
+    return 'Get a chat conversations by id, participants or chat name';
+  }
+
+  public async commandAction(logger: Logger, args: CommandArgs, cb: () => void): Promise<void> {
+
+    try {
+      const chatId = args.options.id
+        || args.options.participants && await this.getChatIdByParticipants(args.options.participants)
+        || args.options.name && await this.getChatIdByName(args.options.name);
+
+      const chat = await this.getChatDetailsById(chatId as string);
+
+      logger.log(chat);
+      cb();
+    }
+    catch(error) {
+      this.handleRejectedODataJsonPromise(error, logger, cb);
+    }
+  }
+
+  public options(): CommandOption[] {
+    const options: CommandOption[] = [
+      {
+        option: '-i, --id [id]'
+      },
+      {
+        option: '-p, --participants [participants]'
+      },
+      {
+        option: '-n, --name [name]'
+      }
+    ];
+
+    const parentOptions: CommandOption[] = super.options();
+    return options.concat(parentOptions);
+  }
+
+  public validate(args: CommandArgs): boolean | string {
+    if (!args.options.id && !args.options.participants && !args.options.name) {
+      return 'Specify id or participants or name, one is required.';
+    }
+
+    let nrOfMutuallyExclusiveOptionsInUse = 0;
+    if (args.options.id) { nrOfMutuallyExclusiveOptionsInUse++; }
+    if (args.options.participants) { nrOfMutuallyExclusiveOptionsInUse++; }
+    if (args.options.name) { nrOfMutuallyExclusiveOptionsInUse++; }
+
+    if (nrOfMutuallyExclusiveOptionsInUse > 1) {
+      return 'Specify either id or participants or name, but not multiple.';
+    }
+
+    if (args.options.id && !validation.isValidTeamsChatId(args.options.id)) {
+      return `${args.options.id} is not a valid Teams ChatId.`;
+    }
+
+    if (args.options.participants) {
+      const participants = args.options.participants.toLowerCase().replace(/\s/g, '').split(',').filter(e => e && e !== '');
+      if (!participants || participants.length === 0 || participants.some(e => !validation.isValidUserPrincipalName(e))) {
+        return `${args.options.participants} contains one or more invalid email addresses.`;
+      }
+    }
+
+    return true;
+  }
+  
+  private async getChatDetailsById(id: string): Promise<Chat> {
+    const requestOptions: AxiosRequestConfig = {
+      url: `${this.resource}/v1.0/chats/${encodeURIComponent(id)}`,
+      headers: {
+        accept: 'application/json;odata.metadata=none'
+      },
+      responseType: 'json'      
+    };
+
+    const chat = await request.get<Chat>(requestOptions);
+    return chat;
+  }
+
+  private async getChatIdByParticipants(participantsString: string): Promise<string> {
+    const participants = participantsString.toLowerCase().replace(/\s/g, '').split(',').filter(e => e && e !== '');
```

Comment:
Let's move converting comma-separated list of users to a separate method that we can share between validation and here to ensure consistency